README.md

```diff
@@ -205,7 +209,16 @@ It needs to compile and provide 100% coverage. A model can only write such tests
 
 On a high level, `DevQualityEval` asks the model to produce tests for an example case, saves the response to a file and tries to execute the resulting tests together with the original source code.
 
-#### Reward Points
+#### Cases
+
+Currently, the following cases are available for this task:
+
+- Java
+  - `plain/src/main/java/plain.java`: An empty function that does nothing.
+- Go
+  - `plain/plain.go`: An empty function that does nothing.
+
+### Reward Points
 
 Currently, the following points are awarded for this task:
```

Comment:
`Currently, the following points are awarded for any task:`

also could you please change the coverage scoring below... it is not based on 100% anymore but what portions of the code is covered